src/webgpu/api/operation/rendering/depth.spec.ts

```diff
@@ -23,6 +221,167 @@ g.test('depth_write_disabled')
+          stencilColor: equalCompareColor,
+          stencilRefValue: 1,
+          stencilColorExpected: false,
+        },
+        {
+          stencilCompare: 'greater',
+          stencilColor: greaterCompareColor,
+          stencilRefValue: 2,
+          stencilColorExpected: true,
+        },
+        {
+          stencilCompare: 'greater-equal',
+          stencilColor: lessCompareColor,
+          stencilRefValue: 0,
+          stencilColorExpected: false,
+        },
+        {
+          stencilCompare: 'greater-equal',
+          stencilColor: equalCompareColor,
+          stencilRefValue: 1,
+          stencilColorExpected: true,
+        },
+        {
+          stencilCompare: 'greater-equal',
+          stencilColor: greaterCompareColor,
+          stencilRefValue: 2,
+          stencilColorExpected: true,
+        },
+        {
+          stencilCompare: 'less',
+          stencilColor: lessCompareColor,
+          stencilRefValue: 0,
+          stencilColorExpected: true,
+        },
+        {
+          stencilCompare: 'less',
+          stencilColor: equalCompareColor,
+          stencilRefValue: 1,
+          stencilColorExpected: false,
+        },
+        {
+          stencilCompare: 'less',
+          stencilColor: greaterCompareColor,
+          stencilRefValue: 2,
+          stencilColorExpected: false,
+        },
+        {
+          stencilCompare: 'less-equal',
+          stencilColor: lessCompareColor,
+          stencilRefValue: 0,
+          stencilColorExpected: true,
+        },
+        {
+          stencilCompare: 'less-equal',
+          stencilColor: equalCompareColor,
+          stencilRefValue: 1,
+          stencilColorExpected: true,
+        },
+        {
+          stencilCompare: 'less-equal',
+          stencilColor: greaterCompareColor,
+          stencilRefValue: 2,
+          stencilColorExpected: false,
+        },
+        {
+          stencilCompare: 'never',
+          stencilColor: lessCompareColor,
+          stencilRefValue: 0,
+          stencilColorExpected: false,
+        },
+        {
+          stencilCompare: 'never',
+          stencilColor: equalCompareColor,
+          stencilRefValue: 1,
+          stencilColorExpected: false,
+        },
+        {
+          stencilCompare: 'never',
+          stencilColor: greaterCompareColor,
+          stencilRefValue: 2,
+          stencilColorExpected: false,
+        },
+        {
+          stencilCompare: 'not-equal',
+          stencilColor: lessCompareColor,
+          stencilRefValue: 0,
+          stencilColorExpected: true,
+        },
+        {
+          stencilCompare: 'not-equal',
+          stencilColor: equalCompareColor,
+          stencilRefValue: 1,
+          stencilColorExpected: false,
+        },
+        {
+          stencilCompare: 'not-equal',
+          stencilColor: greaterCompareColor,
+          stencilRefValue: 2,
+          stencilColorExpected: true,
+        },
+      ])
```

Comment:
nit: `as const` here should avoid the need for `as GPUCompareFunction` below.
```suggestion
      ] as const)
```

Reply:
Done.